### PR TITLE
Update the translation with Denver's review points.

### PR DESCRIPTION
**Pull request by silabs-junF**
Update the translation with Denver's review points.
DOC IMP
**Description about the pull request**
-Update the translation with Denver's review points.
**Review**
@MarkDing  @Denver Lu

### CODE REVIEW

File: hardware/PCB-Layout-Design-Guide-CN.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 本文目的是以实践经验帮助客户用EFR32无线系列芯片设计PCB，以达到较好的射频性能。
```
Comment (silabs-DenverL) [inline hardware/PCB-Layout-Design-Guide-CN.md:26]: To make it consistent with Zigbee document. Suggest change to "概述" from "简介".

File: hardware/PCB-Layout-Design-Guide-CN.md [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
 PCB布线要求在[AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf)里有详细描述。EFR32第一代系列的模块PCB布线在[the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc)里讲述。另外在Simplicity Studio里面可以找到EFR32芯片的参考设计文件。
 
-# 1. 为什么正确PCB布线是很重要的?PCB
```
[the remark this answers is not in the document]
Comment (MarkDing) [reply hardware/PCB-Layout-Design-Guide-CN.md:29]: image

[the remark this answers is not in the document]
Comment (MarkDing) [reply hardware/PCB-Layout-Design-Guide-CN.md:29]: image

File: hardware/PCB-Layout-Design-Guide-CN.md (hunk 1 above)
Comment (silabs-junF) [reply hardware/PCB-Layout-Design-Guide-CN.md:26]: Modified.

File: hardware/PCB-Layout-Design-Guide-CN.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (silabs-junF) [reply hardware/PCB-Layout-Design-Guide-CN.md:29]: Modified.

**Pull request merged**